static/src/javascripts/projects/common/modules/ui/dropdowns.js

```diff
@@ -0,0 +1,35 @@
+// @flow
+import bean from 'bean';
+import fastdom from 'fastdom';
+
+const containerSelector = '.dropdown';
+const buttonCN = 'dropdown__button';
+const contentCN = 'dropdown__content';
+
+const updateAria = (container: Element): void => {
+    const v: boolean = container.classList.contains('dropdown--active');
+    const content = [...container.getElementsByClassName(contentCN)];
+    const button = [...container.getElementsByClassName(buttonCN)];
```

Comment:
I mean it's just `content.concat(button)`. to be frank, I'm not sure it is needed (e.g. there might be only one content and one button), but bonzo abstracts on arrays, so this version is functionally equivalent.